.storybook/preview.js

```diff
@@ -43,6 +43,11 @@ window.loadTimeData = {
     return false
   }
 }
+
+if (!('chrome' in window)) {
+  window.chrome = {}
+}
+
```

Comment:
wait, why is this neccessary for this PR?

Reply:
so, when testing storybook ui on mobile safari i had the following error and it was because of this line `chrome.extension = {...}` - assuming that's only available in chrome browsers where window has a chrome object available?

> ReferenceError: Can’t find variable: chrome

Reply:
Unrelated to this PR, but I think it's a useful fix, and pretty small, so okay to sneak in